cors_filter.go

```diff
@@ -21,7 +21,11 @@ import (
 type CrossOriginResourceSharing struct {
 	ExposeHeaders  []string // list of Header names
 	AllowedHeaders []string // list of Header names
-	AllowedDomains []string // list of allowed values for Http Origin. An allowed value can be a regular expression to support subdomain matching. If empty all are allowed.
+	// AllowedDomains list of allowed values for Http Origin.
+	// An allowed value can be a regular expression to support subdomain matching.
+	// Non-regular expression values will be changed into an exact match: ^yourdomain.com$
```

Comment:
It's not clear to me from this what qualifies as "non-regular expression". `example.com` and `.*` and `example\.com` are all regular expressions, even though they don't start with `^` or end with `$`.